lib/ansible/modules/system/seboolean.py

```diff
@@ -32,6 +32,12 @@
       - Desired boolean value
     type: bool
     required: true
+  ignore_selinux_state:
+    description:
+    - Run independent of selinux runtime state
```

Comment:
Change `selinux` to `SELinux`.

Explain this option a bit more, and when you may want to use it. You can use some of the language from the PR summary.